### PR TITLE
FIX alterationType with entityDelete not working if condition.attrs is empty

### PR DESCRIPTION
**Pull request by fgalan**
Issue #4325

### CODE REVIEW

File: doc/manuals/orion-api.md [hunk 1]
```diff
@@ -2524,7 +2524,10 @@ subscription is triggered. At the present moment, the following alteration types
   and it actually changes (or if it is not an actual update, but `forcedUpdate` option is used
   in the update request)
 * `entityCreate`: notification is sent whenever a entity covered by the subscription is created
-* `entityDelete`: notification is sent whenever a entity covered by the subscription is deleted
+* `entityDelete`: notification is sent whenever a entity covered by the subscription is deleted.
+  In this case, the `attrs` field within [`conditions`](#subscriptionsubjectcondition) is
```
Comment (fgalan) [inline doc/manuals/orion-api.md:2528]:
```suggestion
  In this case, the `attrs` field within [`condition`](#subscriptionsubjectcondition) is
```

**Comment by fgalan** [comment]
@fisuda this PR does a slight modification in documentation. It would be great if you could do a side PR to upgrade Japanese translation (maybe using bug/4325_entityDelete_ignores_attrs branch as base, so it integrates into this PR). Thanks!

**Comment by fisuda** [comment]
I sent the PR #4328. Thanks.

**Review by AlvaroVega — APPROVED** [review]
LGTM

**Pull request merged**